source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -929,8 +929,7 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
        * the time to deliver the `ConnectionCreatedEvent`.
        * Doing so eliminates a thing to worry about in support cases related to this duration,
        * because the time to deliver synchronously is affected by user code.
-       * The driver MUST document this behavior
-       * as well as explicitly warn users that the behavior may change in the future.
+       * We MAY reconsider this decision in the future.
```

Comment:
If the intention that this prose be copied verbatim into API contracts?  If so, it's unusual RFC-ish language to include (i.e. MAY in caps).  The line above about support cases is also quite unusual for API prose.

Reply:
> If the intention that this prose be copied verbatim into API contracts?

No, the intention now is to avoid requiring drivers to document anything, while also not allowing them to commit to a specific behavior. Indeed, this "may" shouldn't be in caps.